packages/ez-vue/tests/unit/recipes/area/AreaChart.spec.tsx

```diff
@@ -11,6 +11,7 @@ describe('AreaChart', () => {
   it('renders an area chart', async () => {
     const wrapper = render(AreaChart, {
       propsData: {
+        onResize: () => undefined,
```

Comment:
@AlanLes I think the most approriate TS return type for this is "void".
```suggestion
        onResize: () => void,
```
Can you do a bulk replace :pray: ?

Reply:
This is not a type return but a value return, it's a function that returns value immediately. And `void` is a type, so it can't fit here :s

Reply:
Oups, right ! Then we're good.